.babelrc

```diff
@@ -1,12 +1,12 @@
 {
-  "presets": [
-    [
-      "@babel/preset-env",
-      {
-        "targets": {
-          "node": true
+    "presets": [
+      [
+        "@babel/preset-env",
+        {
+          "targets": {
+            "node": true
+          }
         }
-      }
+      ]
     ]
-  ]
-}
\ No newline at end of file
+  }
```

Comment:
This is a weird file change -- it's not like it was neatened up for lint exactly, as there's still no newline at end of file or the like. What's this for?